OpenRA.Mods.Common/Orders/PlaceBuildingOrderGenerator.cs

```diff
@@ -65,7 +67,11 @@ public IEnumerable<Order> Order(World world, CPos xy, MouseInput mi)
 				world.CancelInputMode();
 
 			var ret = InnerOrder(world, xy, mi).ToArray();
-			if (ret.Length > 0)
+
+			// If there was a successful placement order
+			if (ret.Any(o => o.OrderString == "PlaceBuilding"
+				|| o.OrderString == "LineBuild"
```

Comment:
I think it's easier to both see how far the expression extends (which is not always obvious at first glance thanks to our indentation rules) and what the operators are. Just makes for easier readability IMO.